stdlib/public/core/ClosedRange.swift

```diff
@@ -488,3 +488,26 @@ extension ClosedRange {
 // shorthand. TODO: Add documentation
 public typealias CountableClosedRange<Bound: Strideable> = ClosedRange<Bound>
   where Bound.Stride : SignedInteger
+
+extension ClosedRange: Codable where Bound: Codable {
+  private enum CodingKeys: String, CodingKey {
+    case lowerBound = "from"
+    case upperBound = "to"
```

Comment:
Let's use terminology consistent with existing facilities: `stride(from:to:by:)` is _exclusive_ of the end, whereas `stride(from:through:by:)` is _inclusive_. Therefore, let's use "through" here.

Reply:
Sure, makes total sense.